.circleci/config.yml

```diff
@@ -136,6 +136,21 @@ commands:
           command: |
             sudo apt-get update -y && sudo apt-get install -y libsnappy-dev zlib1g-dev libbz2-dev liblz4-dev libzstd-dev
 
+  install-libprotobuf-mutator:
+    steps:
+      - run:
+          name: Install libprotobuf-mutator libs
+          command: |
+            git clone --single-branch --branch master --depth 1 git@github.com:google/libprotobuf-mutator.git ~/libprotobuf-mutator
+            cd ~/libprotobuf-mutator && mkdir build && cd build
+            cmake .. -GNinja -DCMAKE_C_COMPILER=clang-13 -DCMAKE_CXX_COMPILER=clang++-13 -DCMAKE_BUILD_TYPE=Release -DLIB_PROTO_MUTATOR_DOWNLOAD_PROTOBUF=ON
+            ninja && sudo ninja install
```

Comment:
Ideally, we should ping libprotobuf to a fixed version instead of master (but seems they only have one master branch).
is there a pre-built package we could install?

Reply:
I searched online but did not find a pre-built package for ubuntu or centos.
The commit history suggests that updates are rare, thus I guess we are fine for now.

Reply:
Looks like it's available on SUSE https://packagehub.suse.com/packages/libprotobuf-mutator/